kaldb/src/main/java/com/slack/kaldb/chunkManager/ChunkManager.java

```diff
@@ -1,6 +1,6 @@
 package com.slack.kaldb.chunkManager;
 
-import static com.slack.kaldb.config.KaldbConfig.LOCAL_QUERY_TIMEOUT_DURATION;
+import static com.slack.kaldb.server.KaldbConfig.LOCAL_QUERY_TIMEOUT_DURATION;
```

Comment:
What's the reason here to change the package of `KaldbConfigs` class?  It seems better suited for the `config` package right?

Reply:
but we literally are using it outside the package :) I don't get the reasoning . Could you please explain more about the change motivation

Reply:
These are static fields. The other methods are package private.